index.js

```diff
@@ -128,7 +128,7 @@ AssetRewrite.prototype.rewriteAssetPath = function (string, assetPath, replaceme
   var self = this;
   return newString.replace(new RegExp('sourceMappingURL=' + escapeRegExp(assetPath)), function(wholeMatch) {
     var replaceString = replacementPath;
-    if (self.prepend && self.prepend !== '' && (!/^sourceMappingURL=(http|https|\/\/)/.test(wholeMatch))) {
+    if (self.prepend && replaceString.indexOf(this.prepend) !== 0 && (!/^sourceMappingURL=(http|https|\/\/)/.test(wholeMatch))) {
```

Comment:
Whoops! Copy/paste error.

I guess I didn't add test coverage for prepending, er, absolute ... to source maps? What is this block doing? :)